.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
You can add `.vscode` to your .gitignore file we usually want to ignore this file.